docs/terminal/williamtown.md

```diff
@@ -14,12 +14,10 @@
 † *Non-standard positions* may only be used in accordance with [VATPAC Air Traffic Services Policy](https://vatpac.org/publications/policies){target=new} 
 
 ## Airspace
-### Default
-By default, WLM TCU owns the airspace within the WLM MIL CTR `SFC` to `F125`. This may be amended by NOTAM.
+WLM TCU owns the airspace within the WLM MIL CTR `SFC` to `F125`.
 
-When WLM TCU is active above `F125` by ad-hoc release or NOTAM, WLM-L_APP owns the airspace `SFC` to `F125`, while WLM APP owns the rest of WLM TCU airspace above `F125`. This may be adjusted as required with coordination.
+When WLM TCU is active above `F125` by ad-hoc release or NOTAM, **WLM-L_APP** owns the airspace `SFC` to `F125`, while **WLM APP** owns the rest of WLM TCU airspace above `F125`. This may be adjusted as required with coordination.
```

Comment:
@alphadelta332 @ST6-Bravo either of you guys know what the upper limit of the WLM TMA is when both positions are online?

Reply:
It's not particularly well documented in the SOPs at the moment, but I don't believe there is a default. It's just whatever additional airspace is released or RAs activated above F125